src/formio.common.ts

```diff
@@ -31,6 +31,36 @@ export interface FormioForm {
   submissionAccess?: AccessSetting[];
 }
 
+export interface GridColumn {
+  label: string;
+  path: string;
+  renderCell?(cellValue: any, component?: any): string;
+}
+
+export interface ComponentInstance {
+  component: ExtendedComponentSchema;
+  getView: any;
+  id: string;
+  type: string;
+  asString?(value: any): string;
+}
+
+export interface GridHeader {
+  component: ComponentInstance | null;
+  key: string;
```

Comment:
small note: if possible for compatibility, I'd rename it to `path`, because if I'm right it actually is always a path to data value, like `data.myKey` instead of just `myKey`